UnitTests/UnitTests.cs

```diff
@@ -29,14 +29,16 @@ public int GivenInputIsASingleNumberWhenAddThenReturnsThatNumber(string input)
         
         [TestCase("1,2", ExpectedResult=3)]
         [TestCase("3,5", ExpectedResult=8)]
-        public int GivenInputIsTwoNumbersWhenAddThenReturnsSumOfNumbers(string input)
+        [TestCase("1,2,3", ExpectedResult=6)]
+        [TestCase("3,5,3,9", ExpectedResult=20)]
+        public int GivenInputIsAmountOfNumbersWhenAddThenReturnsSumOfNumbers(string input)
```

Comment:
Hmmm, basing it off of your other test description, think a better name for this test would be something like _given separated by commas_